tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckMetazones.java

```diff
@@ -39,20 +39,32 @@ public CheckCLDR handleCheck(String path, String fullPath, String value, Options
             // MessageFormat with arguments
         }
 
-        if (path.indexOf("/long") >= 0) {
```

Comment:
@macchiati is there any reason only `/long` had this restriction? Why would a short DST name for a non-DST metazone make any more sense?

Reply:
Can't think of any reason.